packages/ens-controller/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "../../tsconfig.packages.json",
+  "compilerOptions": {
+    "baseUrl": "./"
+  },
+  "references": [
```

Comment:
We must tell TypeScript which other packages this package depends upon, so that it knows to build these packages first before building `ens-controller`.